src/components/PeopleItem/PeopleItem.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {TouchableOpacity, Text, View} from 'react-native';
+import getGlobalStyles from '~/helpers/GlobalStyles';
+import Colors from '../../helpers/Colors';
+import {getInitials} from '../../helpers/Functions';
```

Comment:
Use ~ for both of these and always try to be consistent.

Reply:
Relative imports are due to a misconfiguration in VSCode's auto-import. Fixed manually for now, will update code editor settings soon.